design/proof-network.md

```diff
@@ -0,0 +1,257 @@
+Storage proof network
```

Comment:
I really would like a different name, as the network is most definitely not doing proofs. It's _checking_ proofs.

This name is so misleading that already within the Codex team I've seen it to be referred as a "prover network", while it's the exact opposite of that.

Reply:
I wouldn't mind renaming this, do you have a suggestion? Perhaps "proof checking network"?

Reply:
I don't have a really good name in my mind. "Proof checking network" is certainly one possibility, "Verification network" or "Validator network" could be some more. From these 3 maybe I prefer the last one. But open to other suggestions